dags/examples/maxtext_sweep_gke_example_dag.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+An example DAG to launch a sweep of MaxText GKE XPK workloads
+sweeping over a range of per_device_batch_size values for a 16B model
+on 1xv4-128.
+"""
+
+import datetime
+from airflow import models
+from dags import test_owner
+from dags.vm_resource import TpuVersion, Zone, Project
+from dags.multipod.configs import maxtext_sweep_gke_config
+
+
+with models.DAG(
+    dag_id="maxtext_sweep_gke_example_dag",
+    schedule=None,
+    tags=["multipod_team", "maxtext"],
+    start_date=datetime.datetime(2024, 1, 10),
+    catchup=False,
+) as dag:
+  # MaxText set up and run commands
+  base_run_model_cmds = [
+      "python3 MaxText/train.py MaxText/configs/base.yml base_output_directory=gs://maxtext-experiments-multipod/ dataset_path=gs://max-datasets-rogue enable_checkpointing=false global_parameter_scale=16 steps=10",
+  ]
+
+  # Get list of MaxText GKE XPK jobs
+  maxtext_sweep_gke_test = maxtext_sweep_gke_config.get_maxtext_sweep_gke_config(
+      test_owner=test_owner.RAYMOND_Z,
+      project_name=Project.TPU_PROD_ENV_MULTIPOD.value,
+      cluster_name="v4-bodaborg",
```

Comment:
Nit: we could reuse this variable: https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/dags/vm_resource.py#L69